bounce/server/api/users.py

```diff
@@ -9,10 +9,12 @@
 from ...db import image, user
 from ...db.image import EntityType
 from ..resource import validate
-from ..resource.user import GetUserResponse, PostUsersRequest, PutUserRequest
+from ..resource.user import (GetUserResponse, PostUsersRequest, PutUserRequest,
+                             SearchUsersRequest, SearchUsersResponse)
 
 # Maximum number of bytes in an image upload
 IMAGE_SIZE_LIMIT = 1000000
+MAX_SIZE = 20
```

Comment:
I think it would be better to import this variable from `db/user.py` rather than redefining it.